src/mixer/basetrackplayer.cpp

```diff
@@ -2,6 +2,7 @@
 
 #include <QMessageBox>
 #include <QMetaMethod>
+#include <QString>
```

Comment:
this is not required, this class already uses QString in many places

Reply:
OK, thank you

Reply:
I thought 'resolve' meant see whole conversation.....  sorry